lib/rules/no-unmodified-loop-condition.js

```diff
@@ -261,10 +261,10 @@ module.exports = function(context) {
 
             for (var j = 0; j < condition.groups.length; ++j) {
                 var be = condition.groups[j];
-                var group = groupMap.get(be);
+                var group = groupMap[be];
```

Comment:
Thats fine. I am only saying that if this is a relevant use case, then there should be a test that would cover it. I will re-implement it using `_.find` and an array of collections.